pycbc/waveform/waveform.py

```diff
@@ -822,6 +823,13 @@ def imrphenomd_length_in_time(**kwds):
 from . nltides import nonlinear_tidal_spa
 cpu_fd["TaylorF2NL"] = nonlinear_tidal_spa
 
+# Load external waveforms #####################################################
+if 'PYCBC_WAVEFORM' in os.environ:
+    mods = os.environ['PYCBC_WAVEFORM'].split(':')
+    for mod in mods:
+        mhandle = __import__(mod, fromlist=[''])
+        mhandle.add_me(cpu_fd, cpu_td)
```

Comment:
Should we expose the "additional" information for waveforms? E.g. without `_filter_time_lengths` we couldn't use this for some hypothetical optimized waveform filter.

Reply:
Yes, I agree, that makes sense. I'll add that before taking this back out of WIP.